HACKING.rst.txt

```diff
@@ -380,6 +380,19 @@ test and use your changes (in ``.py`` files), by simply restarting the
 interpreter.
 
 
+*Are there any video examples for installing from source, setting up a 
+development environment, etc...?*
+
+`Anaconda SciPy Dev Part I`_ is a four-minute
+overview of installing Anaconda, building SciPy from source, and testing
+changes made to SciPy from the Spyder IDE.
```

Comment:
Maybe it would be worth mentioning here on what OS you're building? From my understanding the setup might differ significantly on different systems and mentioning that early might prevent confusion. Furthermore it would make it easier to discover, e.g. searching for "macOS".

Reply:
Absolutely. Not sure how it slipped my mind.